ext/node/polyfills/worker_threads.ts

```diff
@@ -32,6 +32,67 @@ export interface WorkerOptions {
   workerData?: unknown;
 }
 
+const WHITESPACE_ENCODINGS: Record<string, string> = {
```

Comment:
I'm not sure where to put all of this `toFileUrl` code. I considered putting it in https://github.com/denoland/deno/blob/main/ext/node/polyfills/_utils.ts but the dependency of this code on `node/path`'s `isAbsolute` function complicates the file dependencies as then everything that includes that file now transitively depends on `node/path`.

For now I kept it in this file, but would like other input on where this logic might best live.

Reply:
Putting these here looks fine to me